packages/app/ios/Orbit/IngestEventEmitter.m

```diff
@@ -0,0 +1,78 @@
+//
+//  EventEmitter.m
+//  Orbit
+//
+//  Created by Ozzie Kirkby on 2022-05-07.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "IngestEventEmitter.h"
+#import <libkern/OSAtomic.h>
+#import <stdatomic.h>
+
+@implementation IngestEventEmitter
+
+@synthesize semaphore;
+
+- (instancetype)init {
```

Comment:
Shouldn't be necessary, since you're not doing anything other than calling super.